src/CoreFoundation/OSLog.cs

```diff
@@ -30,8 +30,15 @@ namespace CoreFoundation {
 	[Introduced (PlatformName.MacCatalyst, 13, 0)]
 	public sealed class OSLog : NativeObject {
 
-		// initialized only once (see tests/cecil-tests/)
-		public static OSLog Default { get; } = new OSLog (IntPtr.Zero, false);
+		static IntPtr _os_log_default;
+
+		static OSLog ()
+		{
+			_os_log_default = Dlfcn.dlsym (Libraries.System.Handle, "_os_log_default");
```

Comment:
Would make more sense to fetch this on demand in the Default getter? Otherwise it can't be linked away if someone doesn't use the Default property (it's possible to create custom OSLogs).

Reply:
yes, another reason is that this will run into a similar issue (TLE) on older OS (even if less a problem since it would be unusable if called anyway)